### PR TITLE
fix: accessibility improvements in UiScale component

### PR DESCRIPTION
**Pull request by jbucior**
## Description
- Add/change aria-labels
- Add/change aria-labelledby
- Add aria-checked and role radio to UiRadio element
- Change default min and max value to more generic

## Related Issue
https://infermedica.atlassian.net/wiki/spaces/DS/pages/4086923423/Scale

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by pspaczek** [comment]
@jbucior please add the issue. Because I don't know why we need these changes.

**Review by sonya0504 — DISMISSED** [review]

File: src/components/molecules/UiScale/UiScale.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@
       >
         <legend
           v-if="legend"
+          id="scale-legend"
```
Comment (pspaczek) [inline src/components/molecules/UiScale/UiScale.vue:19]: @jbucior `id` should be unique what about when dev put 2 `UiScale` on view ;) 
IMO should be 
```
const scaleId = computed(() => (
  props.id || `scale-${uid()}`
));

Comment (pspaczek) [inline src/components/molecules/UiScale/UiScale.vue:19]:
```suggestion
          :id="scaleId"
```

File: src/components/molecules/UiScale/UiScale.vue [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <div
     class="ui-scale"
     role="radiogroup"
-    :aria-label="defaultProps.translation.label"
+    :aria-labelledby="legend ? 'scale-legend' : undefined"
```
Comment (pspaczek) [inline src/components/molecules/UiScale/UiScale.vue:6]:
```suggestion
    :aria-labelledby="scaleId"
```

File: src/components/molecules/UiScale/UiScale.vue [hunk 3]
```diff
@@ -31,7 +32,8 @@
           v-model="scaleValue"
           :value="index"
           class="ui-scale__option"
-          :aria-labelledby="`scale-label-${index}`"
+          :aria-checked="scaleValue === index"
+          role="radio"
```
Comment (pspaczek) [inline src/components/molecules/UiScale/UiScale.vue:36]: ? You get `<label role="radio" aria-checked="true"><input type="radio/></label>`. IMO it's not expected result. And I found that https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked

> Note: Where possible use an HTML [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) element with type="checkbox" as this element has built in semantics and does not require ARIA attributes.

**Review by pspaczek — CHANGES_REQUESTED** [review]
Please check my comment @jbucior

File: src/components/molecules/UiScale/UiScale.vue (hunk 1 above)
Comment (jbucior) [reply src/components/molecules/UiScale/UiScale.vue]: Removed

**Review by pspaczek — APPROVED** [review]

**Pull request merged**